src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx

```diff
@@ -141,23 +130,11 @@ export default class PatientViewPathwayMapper extends React.Component<
         });
         return keyed_genes;
     }
-    @computed get isNewStoreReady() {
-        return (
-            this.storeForAllData &&
-            this.storeForAllData.samples.isComplete &&
-            this.storeForAllData.mergedMutationData &&
-            this.storeForAllData.coverageInformation.isComplete &&
-            this.storeForAllData.mergedDiscreteCNADataFilteredByGene &&
-            this.storeForAllData
-                .mergedMutationDataIncludingUncalledFilteredByGene
-        );
-    }
     public render() {
         //control the data
-        if (this.isNewStoreReady) {
-            this.addGenomicData(this.alterationFrequencyData);
-            this.getQueryGenes(this.alterationFrequencyData);
-        }
+        this.alterationFrequencyDataForQueryGenes;
+        this.getQueryGenes(this.alterationFrequencyData);
```

Comment:
I don't think these calls are necessary. We already refer them later in the flow when we pass props to `this.PathwayMapperComponent`. Also we should remove the `control the data` comment above.